python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
+    tap:column_index: 1192
   - name: u_gaapFlux_flag_gaussianization
     "@id": "#Object.u_gaapFlux_flag_gaussianization"
     datatype: boolean
     description: PSF Gaussianization failed when trying to scale by this factor. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1193
   - name: u_gaapOptimalFlux
     "@id": "#Object.u_gaapOptimalFlux"
     datatype: double
     description: GaaP flux with optimal aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1194
   - name: u_gaapOptimalFluxErr
     "@id": "#Object.u_gaapOptimalFluxErr"
     datatype: double
     description: GaaP flux uncertainty with optimal aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1195
   - name: u_gaapOptimalFlux_flag_bigPsf
     "@id": "#Object.u_gaapOptimalFlux_flag_bigPsf"
     datatype: boolean
     description: The Gaussianized PSF is bigger than the aperture. Forced on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1196
   - name: u_gaapPsfFlux
     "@id": "#Object.u_gaapPsfFlux"
     datatype: double
     description: GaaP flux with PSF aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1197
   - name: u_gaapPsfFluxErr
     "@id": "#Object.u_gaapPsfFluxErr"
     datatype: double
     description: GaaP flux uncertainty with PSF aperture after multiplying the seeing by 1.15. Forced on u-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 1198
   - name: u_hsmShapeRegauss_e1
     "@id": "#Object.u_hsmShapeRegauss_e1"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1199
   - name: u_hsmShapeRegauss_e2
     "@id": "#Object.u_hsmShapeRegauss_e2"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1200
   - name: u_hsmShapeRegauss_flag
     "@id": "#Object.u_hsmShapeRegauss_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1201
   - name: u_hsmShapeRegauss_sigma
     "@id": "#Object.u_hsmShapeRegauss_sigma"
     datatype: double
     description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1202
   - name: u_iDebiasedPSF_flag
     "@id": "#Object.u_iDebiasedPSF_flag"
     datatype: boolean
     description: General failure flag, set if anything went wrong. Measured on u-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 1203
```

Comment:
This "Debiased" flag should go with the actual debiased PSF shape columns below.  Same with the "Round" one.

I have no idea what the "i" prefix in these columns means.  I wonder if it's a typo, or if somebody took the convention of writing `I_{xx}` (etc.) for image moments the wrong way.